db/migrate/20200317162403_remove_disability_compensation_job_statuses.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDisabilityCompensationJobStatuses < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :disability_compensation_job_statuses
```

Comment:
I don't really like using change here because the automagic down would be to create the table. This doesn't actually reverse the effects of dropping a table because creating the table doesn't reinstate dropped data, dropping a table is in reality irreversible. So, I originally wanted to writeup/down functions instead and have my down raise ActiveRecord::IrreversibleMigration but that kind of stinks too because there isn't any data in this table or the disability_compensation_submissions table in prod-- it seems silly to fail rollbacks for this. In summation, all these solutions are imperfect, so I picked the least controversial way forward.

Reply:
I get the concern and appreciate the thought -- but since there's no table structure here, what do you get when you reverse the `drop_table`. I honestly never thought about it

Reply:
Actually figured it out

```
$ rake db:migrate
Running via Spring preloader in process 30545
/Users/johnpaul/work/oddball/vets-api/rakelib/connectivity.rake:11: warning: already initialized constant REDIS_CONFIG
/Users/johnpaul/work/oddball/vets-api/config/initializers/redis.rb:4: warning: previous definition of REDIS_CONFIG was here
== 20200317162403 RemoveDisabilityCompensationJobStatuses: migrating ==========
-- drop_table(:disability_compensation_job_statuses)
   -> 0.0026s
== 20200317162403 RemoveDisabilityCompensationJobStatuses: migrated (0.0061s) =

== 20200317164309 RemoveDisabilityCompensationSubmissions: migrating ==========
-- drop_table(:disability_compensation_submissions)
   -> 0.0008s
== 20200317164309 RemoveDisabilityCompensationSubmissions: migrated (0.0035s) =

johnpaul@buffy:~/work/oddball/vets-api (sj-remove-VA526ez-vgt6822-part2) $ rake db:rollback
Running via Spring preloader in process 30689
/Users/johnpaul/work/oddball/vets-api/rakelib/connectivity.rake:11: warning: already initialized constant REDIS_CONFIG
/Users/johnpaul/work/oddball/vets-api/config/initializers/redis.rb:4: warning: previous definition of REDIS_CONFIG was here
== 20200317164309 RemoveDisabilityCompensationSubmissions: reverting ==========
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:



To avoid mistakes, drop_table is only reversible if given options or a block (can be empty).
```

Reply:
I don't love that it also errors.  Do you want me to to do an explicit up/down? I can make down can have the table creation with the col definitions.

Reply:
I think explicit up/down is easier to read. The alternative is 

```
  def change
    drop_table :disability_compensation_job_statuses, id: :serial, force: :cascade  do |t|
      t.datetime "created_at", null: false
      t.datetime "updated_at", null: false
      t.integer "disability_compensation_id"
      t.integer "va526ez_submit_transaction_id"
      t.boolean "complete", default: false
    end
  end
```
which i think isn't really straightforward to understand (why would a drop_table take a block?.... ooooooohhhhh for a rollback)